src/newrelic/processor.go

```diff
@@ -818,10 +823,13 @@ func (p *Processor) doHarvest(ph ProcessorHarvest) {
 	if p.cfg.AppTimeout > 0 && app.Inactive(p.cfg.AppTimeout) {
 		log.Infof("removing %q with run id %q for lack of activity within %v",
 			app, id, p.cfg.AppTimeout)
-		if len(p.apps) == limits.AppLimitNotifyLow {
+		numapps := len(p.apps)
+		if numapps == limits.AppLimitNotifyLow {
 			log.Infof("current number of apps is %d",
 				limits.AppLimitNotifyLow)
 		}
+		log.Healthf("current number of apps is %d of a max of %d",
```

Comment:
This line will double log when numapps == limites.AppLimitNotifyLow. Do we want this, do we want to change the Infof log a little, or do we want the Healthf log in an else?

Reply:
The existing ```Infof ```output could potentially be in use by some customers so I was not comfortable removing it.  It was added to service a similar request from a customer but I don't know they would want to transition to the 'healthcheck' log level due to the considerably higher potential log output.